### PR TITLE
Pa footer

### PR DESCRIPTION
**Pull request by Lt-Kartik-9PARA-SF**

### CODE REVIEW

File: src/components/AppLayout/Footer.tsx [hunk 1]
```diff
@@ -46,84 +29,163 @@ const Footer = ({ className } : { className?:string }) => {
 							</Link>
 						</div>
 
-						<p className="mx-auto mt-6 max-w-md text-center leading-relaxed text-sidebarBlue sm:mx-0 sm:max-w-xs sm:text-left">
-							Polkassembly is a unified platform to discuss and vote on governance proposals, motions and referandas.
-						</p>
+						<div className='mt-3 flex justify-center md:inline-block max-[650px]:hidden'>
+							<Space size={19} className='items-center '>
+								{
```
Comment (alphainfinitus) [inline src/components/AppLayout/Footer.tsx:34]: we don't need these curly braces here

File: src/components/AppLayout/Footer.tsx [hunk 2]
```diff
@@ -46,84 +29,163 @@ const Footer = ({ className } : { className?:string }) => {
...
 										</a>
 									</div>
 								</div>
 							</nav>
 						</div>
 
-						<div className="mt-10 md:mt-0 md:ml-10 lg:ml-14 text-center sm:text-left">
-							<p className="text-lg font-medium text-sidebarBlue">Our Services</p>
+						<div className="mt-8 md:mt-0 md:ml-10 lg:ml-14 text-center sm:text-left text-[#485F7D] font-normal">
+							<p className="text-lg font-bold text-[#243A57] mb-[-5px] max-[650px]:mt-[-15px] ">Our Services</p>
```
Comment (alphainfinitus) [inline src/components/AppLayout/Footer.tsx:99]: please remove arbitrary colour values

File: src/components/AppLayout/Footer.tsx [hunk 3]
```diff
@@ -46,84 +29,163 @@ const Footer = ({ className } : { className?:string }) => {
...
 								</div>
 							</nav>
 						</div>
 					</div>
 				</div>
 
 				{/* Below divider */}
-				<div className="mt-12 border-t border-gray-100 pt-6">
+				<Divider className='mb-0' />
+				<div className="mt-5 pb-3 text-sm text-[#485F7D] font-medium">
```
Comment (alphainfinitus) [inline src/components/AppLayout/Footer.tsx:129]: please remove arbitrary text colour

File: src/components/AppLayout/Footer.tsx [hunk 4]
```diff
@@ -46,84 +29,163 @@ const Footer = ({ className } : { className?:string }) => {
...
+
+						<p className="mb-0">
 							<span className="block sm:inline">All rights reserved.</span>
 						</p>
+					</div>
+				</div>
+				<div className='md:hidden'>
+					<div className='mt-1 flex justify-center'>
+						<Space size={19} className='items-center '>
+							{
```
Comment (alphainfinitus) [inline src/components/AppLayout/Footer.tsx:148]: do not need curly braces here

File: src/components/AppLayout/Footer.tsx [hunk 5]
```diff
@@ -46,84 +29,163 @@ const Footer = ({ className } : { className?:string }) => {
...
+						Polka Labs Private Limited {new Date().getFullYear()}
+							</p>
+						</div>
+
+						<p className="mb-0">
 							<span className="block sm:inline">All rights reserved.</span>
 						</p>
+					</div>
+				</div>
+				<div className='md:hidden'>
```
Comment (alphainfinitus) [inline src/components/AppLayout/Footer.tsx:145]: we don't need whole another  div for small devices, can be done in the div above

**Review by alphainfinitus — CHANGES_REQUESTED** [review]
Please check comments.

**Review by KanishkaRajputd — APPROVED** [review]

**Pull request merged**